tests/Tools/vmOpsTool/InMemoryVMWareImplPlatformTests.java

```diff
@@ -1,12 +0,0 @@
-public class InMemoryVMWareImplPlatformTests extends VMWarePlatformTests {
-
-    @Override
-    public IVMWare getVmWareImpl() {
-        return new InMemoryVMWareImpl();
-    }
-
-    @Override
-    public String getvCenterUrl() {
-        return "http://localhost:8080/sdk/vimservice";

```

Comment:
This url is not actual vCenter url it is just to inject mocking mechanism into testing.